eya_def_tools/eya_def_tools/data_models/wind_resource.py

```diff
@@ -90,15 +85,14 @@ class WindResourceResults(EyaDefBaseModel):
             "measurement location(s), as dimensionless values. The "
             "first standard result dataset should comprise the joint "
             "wind speed and direction probability distributions, with "
-            "dimensions 'wind_dataset_id', 'location_id' (where the "
```

Comment:
This is now simpler when we have only `wind_dataset_id` and not the possibility of also a `location_id`.